Updated and added to potential validation section

RaduV-Dev opened the pull request:
Added two new lines, added to an existing line and modified the XML file size limit.

RaduV-Dev committed:
Updated and added to potential validation section

Added two new lines, added to an existing line and modified the XML file size limit.

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

JohanFreelancer9 commented on microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md:
```diff
@@ -428,9 +428,13 @@ When you upload your rule package XML file, the system validates the XML and che
+
+- When using the PowerShell Cmdlet there is a maximum return size of the Deserialized Data of approximately 1 megabyte.   This will affect the size of your rule pack XML file. Keep the uploaded file limited to a 770 kilobyte maximum as a suggested limit for consistent results without error when processing.
+
+- The XML structure does not require formatting characters such as Spaces, Tabs or Carriage Return / Linefeed entries.  Take note of this when optimizing for space on uploads. Tools such as Microsoft Visual Code provide join line features to compact the XML file.
```
```suggestion
- The XML structure does not require formatting characters such as spaces, tabs, or carriage return/linefeed entries.  Take note of this when optimizing for space on uploads. Tools such as Microsoft Visual Code provide join line features to compact the XML file.
```

JohanFreelancer9 reviewed (changes requested):
Copy review.

yogkumgit commented:
@RaduV-Dev

RaduV-Dev committed:
Update microsoft-365/compliance/create-a-custom-sensitive-information-type-in-scc-powershell.md

Looks good. All I wanted to do is call out the rule pack XML file should be saved as a single line of text in order to maximize the size of it.

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

yogkumgit commented:
@JohanFreelancer9
